src/syslog_mcp/tools/analysis/performance_analyzer.py

```
@@ -0,0 +1,454 @@
+        elif count < mean * 0.1 and count > 0:
+            anomalies.append({
+                "type": "DROP",
+                "timestamp": period["timestamp"], 
+                "count": count,
+                "factor": round(count / mean, 1) if mean > 0 else 0
+            })
+    
+    return anomalies
+
+
+def _analyze_error_trends(error_timeline: List[Dict], hours: int) -> Dict[str, Any]:
```

Ruff (0.12.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)